packages/cli/bin/merkur.mjs

```diff
@@ -28,6 +31,36 @@ const inspectOption = new Option('--inspect', 'Debugging widget server');
 const verboseOption = new Option('--verbose', 'Verbose mode which show debug information.');
 const analyzeOption = new Option('--analyze', 'Analyze bundle.');
 
+let customCommands = [];
+
+// Extend commands from directory
+const extendCommandsFromDir = async commandsDir => {
+  customCommands = customCommands.concat(
+    fs.readdirSync(commandsDir)
+      .map(command => ({ dir: commandsDir, command }))
+      .filter(
+        ({ command, dir }) => !customCommands.some(({ command: existingCommand }) => existingCommand === command) &&
```

Comment:
I feel like we should print at least a warning, that we are ignoring some commands due to using the same file name.  Or throw an error. Or just try to load everything and hope for the best... but I am not sure what would `commander` do if you try to load a command with the same name twice... maybe it would throw an error so we don't really have to deal with this at all and really just load everything?

Right now, if we have

```
@merkur/my-plugin-1
  - lib
    - commands
      - my-command.js

@merkur/my-plugin-2
  - lib
    - commands
      - my-command.js
```

In this case, only the command from @merkur/my-plugin-1 will be loaded and @merkur/my-plugin-2 will be silently ignored, which can cause some confusion.